index.html

```diff
@@ -223,26 +223,30 @@ <h2 id="contact-me">Contact me </h2>
                 <p id="contact-text">
                     If you ave an application you are interested in developing, a feature that you
                     need built or a project that needs coding. I'd love to help with it.
+                    
                 </p>
-                <form action="https://formspree.io/f/mlezoknl" method="POST">
+                <form id="contact_form" action="https://formspree.io/f/mlezoknl" method="POST">
                     <ul id="form-data">
                         <li class="input-data">
-                            <input type="text" name="name" maxlength="30" placeholder="Mirwais Farahi" required />
+                            <input type="text" id="full_name" name="name" maxlength="30" placeholder="Mirwais Farahi"
+                                required />
                         </li>
                         <li class="input-data">
-                            <input type="email" name="email" placeholder="mirfarahi92@gmail.com" required />
+                            <input type="email" id="email_address" name="email" placeholder="mirfarahi92@gmail.com" required />
                         </li>
                         <li class="input-data">
-                            <textarea name="message" maxlength="500" placeholder="Write your message here"
-                                required></textarea>
+                            <textarea name="message" id="user_message" maxlength="500"
```

Comment:
- [ ] [Optional] It would be better to add a little bit of padding in the inputs and the text area.